ui/src/types/buckets.ts

```diff
@@ -1,12 +1,6 @@
-export interface Bucket {
-  id: string
-  name: string
-  organization: string
-  orgID: string
-  rp?: string
-  retentionRules: RetentionRule[]
-  links: BucketLinks
-}
+import {IBucket} from '@influxdata/influx'
+
+export interface Bucket extends IBucket {}
```

Comment:
couldnt IBucket just be used everywhere rather than this interface that doesnt add anything?

Reply:
I prefer us to use src/types instead of a mixture of from here and there.